### PR TITLE
Feature: Episode rich text description

### PR DESCRIPTION
**Pull request by SergioEstevao**
| 📘 Part of: #2479  |  
|:---:|

Fixes # 

This PR adds support for rich html description for podcasts

| 1 | 2 | 3 | 4 |
| - | - | - | - |
| Simulator Screenshot - iPhone 16 Pro - 2024-12-11 at 10 46 49 | Simulator Screenshot - iPhone 16 Pro - 2024-12-11 at 10 46 31 | Simulator Screenshot - iPhone 16 Pro - 2024-12-11 at 10 18 24 |  Simulator Screenshot - iPhone 16 Pro - 2024-12-11 at 10 16 59 |

## To test

1. Start the app using the **staging** environment.
2. Ensure that you have the `usePodcastHTMLDescription` FF enabled.
3. Open the Discovery tab
4. Open a podcast with rich html descriptions. Ex: DirtBag Rich
5. Check if the description shows correctly. This is: links are show with underline, italic show in the last paragraph.
6. Detect if you can tap on the description links and it will open Safari with the URL
7. Tap on the description to expand 
8. Tap on the top arrow to collapse the description
9. Tap on the top arrow to expand the description
10. Change tab
11. Go back to the discovery tab
12. Check if description is showing correctly
13. Another podcast to check is The Daily, this description does not have html but we are replacing the `\n` for `<br>` 
14. Open the podcast description and check if the different paragraphs have a empty line separating them.

## Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by SergioEstevao** [comment]
Version `7.79` has now entered code-freeze, so the milestone of this PR has been updated to `7.80`.

File: podcasts/ExpandableLabel.swift [hunk 1]
```diff
@@ -32,7 +32,36 @@ class ExpandableLabel: ThemeableLabel {
...
+        }
+        let result = try? NSAttributedString(data: data, options: [.documentType: NSAttributedString.DocumentType.html, .characterEncoding: NSUTF8StringEncoding], documentAttributes: nil)
+        attributedText = result
+        textColor = AppTheme.colorForStyle(style)
+        collapsed = linesRequired() > maxLines
+    }
+
+    @objc private func labelTapped(gesture: UITapGestureRecognizer) {
+        if let url = gesture.didTapLinkInLabel(label: self) {
+            UIApplication.shared.open(url)
```
Comment (danielebogo) [inline podcasts/ExpandableLabel.swift:63]: Should we open this externally or internally?

**Review by danielebogo — CHANGES_REQUESTED** [review]
| New | Old |
| -------- | ------- |
| RocketSim_Screenshot_iPhone_16_6 1_2024-12-11_18 07 44 | RocketSim_Screenshot_iPhone_16_6 1_2024-12-11_18 08 07 |

@SergioEstevao I as able to test this in Staging and I see the font size is different. The new attributed string uses a bigger one. Also not sure about redirecting the users out when tapping on the URL. Wdyt?

File: podcasts/ExpandableLabel.swift (hunk 1 above)
Comment (SergioEstevao) [reply podcasts/ExpandableLabel.swift:63]: I followed the same pattern that we have on the page. If you scroll a bit down you will see a section with the podcast url and if you tap on it you are taken to the external browser

**Comment by SergioEstevao** [comment]
@danielebogo just adjusted the font size and commented about the URL opening, can you review it again?

**Review by danielebogo — CHANGES_REQUESTED** [review]
Hey @SergioEstevao sorry to be super picky but the `1.2em` looks smaller than the other one.

I made some test and the closest size I can get with the `em` is `1.34`


| Original | 1.2em |
| -------- | ------- |
| RocketSim_Screenshot_iPhone_16_6 1_2024-12-12_13 47 09 | RocketSim_Screenshot_iPhone_16_6 1_2024-12-12_13 47 36 |


| Original | 1.34em |
| -------- | ------- |
| RocketSim_Screenshot_iPhone_16_6 1_2024-12-12_13 47 09 | RocketSim_Screenshot_iPhone_16_6 1_2024-12-12_13 58 53 |


Would be great to have feedback from @david-gonzalez-a8c  and @dcamozzi

**Review by danielebogo — APPROVED** [review]
Thanks for addressing my feedback!

**Pull request merged**